app/code/Magento/Customer/Model/Address.php

```diff
@@ -400,6 +400,7 @@ private function getAttributeList()
      */
     public function getAttributeSetId()
     {
+        //phpstan:ignore Call to an undefined static method
```

Comment:
As I understood - there is no such method in parent. It have to be replaced with getData()